barrymcgee/snapweb
# Refactor Settings & Store templates for v1

## Done
- Refactored templates to use Vanilla v1 styles
- Swapped out Ubuntu logos to use dark version in designs
- Added active nav border-bottom
- Added overlay cursor to Settings menu
- Removed redundant p-footer override
## QA
- Pull down code
- Re-run snapweb build
- View locally and cross ref with designs

**Note:** This is a first pass and so structure should reflect rough outline of designs. Second pass will finesse.

Store mob: https://github.com/CanonicalLtd/snapweb-design/blob/master/Store/320px/03-snappy-store-card-mobile-320-v3.png

Store desktop: https://github.com/CanonicalLtd/snapweb-design/blob/master/Store/1440px/05-snappy-store-list-1440-card-view-v6.png
## 

Settings Mob: https://github.com/CanonicalLtd/snapweb-design/blob/master/Settings/320px/02-snappy-settings-device-info-320.png

Settings desktop: https://raw.githubusercontent.com/CanonicalLtd/snapweb-design/master/Settings/1440px/00a-snappy-settings-devices-1440.png?token=AAe24rxCY3iG4DvxMKN8FMKxsDmnlvwfks5YExHYwA%3D%3D
